tools/ci_build/github/azure-pipelines/nuget/templates/test_win.yml

```diff
@@ -11,12 +11,15 @@ jobs:
   - NuGet_Packaging
   condition: succeeded()
   variables:
-  - name: OnnxRuntimeBuildDirectory
-    value: '$(Build.BinariesDirectory)'
-  - name: SKIPNONPACKAGETESTS
-    value: 'ON'
-  - name: runCodesignValidationInjection
-    value: false
+    OnnxRuntimeBuildDirectory: '$(Build.BinariesDirectory)'
+    SKIPNONPACKAGETESTS: 'ON'
+    runCodesignValidationInjection: false
+    NugetArtifact: $(Build.BinariesDirectory)\nuget-artifact
+    RunTestScript: test\Microsoft.ML.OnnxRuntime.EndToEndTests\runtest.bat
+    DotNetCoreX64Args: ${{ variables.NugetArtifact }} netcoreapp2.1 x64 $(NuGetPackageVersionNumber)
+    DotNetCoreX86Args: ${{ variables.NugetArtifact }} netcoreapp2.1 x86 $(NuGetPackageVersionNumber)
+    DotNetFrameworkX64Args: ${{ variables.NugetArtifact }} net461 x64 $(NuGetPackageVersionNumber)
```

Comment:
Why do we need to support net461?

Reply:
Previous code has support for net416 as well
https://github.com/microsoft/onnxruntime/blob/master/tools/ci_build/github/azure-pipelines/nuget/templates/test_win.yml#L87

Reply:
Thank you